src/charts/vertical_bar.coffee

```diff
@@ -344,7 +354,10 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
     # the origin line so that they do not overlap with it
     zeroDisplacement = 1
     yScale = @get 'yScale'
-    class: (d,i) -> "grouping-#{i}"
+    labelIDMapping = @get 'labelIDMapping'
+    class: (d,i) ->
+      id = labelIDMapping[d.label]
+      "grouping-#{id}"
```

Comment:
I think you might be able to make this more concise. Something like `id = @get('labelIDMapping')[d.label]`?